packages/html-reporter/src/labels.tsx

```diff
@@ -44,7 +44,10 @@ export const ProjectAndTagLabelsView: React.FC<{
   useLinks?: boolean,
   style?: React.CSSProperties,
 }> = ({ projectNames, activeProjectName, otherLabels, useLinks, style }) => {
-  return (projectNames.length > 0 || otherLabels.length > 0) && <span className='label-row' style={style ?? {}}>
+  // We can have an empty project name if we have no projects specified in the config
```

Comment:
We can also have multiple empty projects. Will this code work in such a case?

```
projects: [
  { name: 'chromium' },
  { /* some settings */ },
  { /* different settings */ },
  { name: 'webkit' },
]
```

Reply:
<img width="506" height="319" alt="Screenshot 2025-08-26 at 5 46 02 AM" src="https://github.com/user-attachments/assets/9d19508b-b23d-4678-9a91-1f1f8aa06fcb" />

It functions with any project without a name.